util/src/main/java/com/google/cloud/hadoop/util/AbstractGoogleAsyncWriteChannel.java

```diff
@@ -145,7 +145,7 @@ private void addBytesToCumulativeChecksum(
     ByteBuffer duplicateBuffer = src.duplicate();
     duplicateBuffer.position(originalPosition);
     // Only calculate hash for written bytes.
```

Comment:
may wanna update comment regarding why position+writtenBytes and not just writtenBytes